_includes/scripts.html

```diff
@@ -39,7 +39,13 @@
 });
 </script>
 
-<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB48gZNKOtEbslmLkTWtphkzCJVM6ITd1s&callback=initialize"></script>
+<script
+  src="https://developers.google.com/maps/documentation/javascript/examples/markerclusterer/markerclusterer.js"
```

Comment:
@swinton, I've been thinking the same, but also been using the same URL ~5 years ago 🙄